### PR TITLE
FIXED - Share Modal: Close icon not adapting to dark mode

### PR DESCRIPTION
**Pull request by subhajit20**
Fixed  Close icon not adapting to dark mode 
Issue No - https://github.com/ToolJet/ToolJet/issues/7315

### CODE REVIEW

File: frontend/src/Editor/ManageAppUsers.jsx [hunk 1]
```diff
@@ -246,7 +251,9 @@ class ManageAppUsersComponent extends React.Component {
                     <div className="input-group">
                       <Textarea
                         disabled
-                        className={`input-with-icon ${this.props.darkMode && 'text-light'}`}
+                        className={`input-with-icon ${this.props.darkMode && 'bg-dark'} ${
+                          this.props.darkMode && 'text-light'
```
Comment (Ckbhatia) [inline frontend/src/Editor/ManageAppUsers.jsx:255]: As you are repeating the destructuring `this.props.darkMode` more than two times. You should destructure the `darkMode` outside JSX and use it.

Comment (subhajit20) [reply frontend/src/Editor/ManageAppUsers.jsx:255]: Okay with do that

**Comment by subhajit20** [comment]
> Please share screen recording or screenshots as well
Screenshot 2023-08-27 at 10 16 06 PM

File: frontend/src/Editor/ManageAppUsers.jsx (hunk 1 above)
Comment (subhajit20) [reply frontend/src/Editor/ManageAppUsers.jsx]: > As you are repeating the destructuring `this.props.darkMode` more than two times. You should destructure the `darkMode` outside JSX and use it.

Hey! done that

**Pull request closed without merging**